uaaextras/webapp.py

```diff
@@ -742,6 +742,7 @@ def reset_totp():
         g.totp.unset_totp_seed(username)
         g.uaac.invalidate_tokens(decoded_token['user_id'])
         session.clear()
+        return redirect(app.config['UAA_BASE_URL'] + '/logout.do')
         return render_template("reset_totp.html", login_link=app.config['UAA_BASE_URL'], reset_complete=True)
```

Comment:
Should this second `return` statement be removed now that the `redirect` is supplied above?

Reply:
:doh: yes